connman/vpn/plugins/openvpn.c

```diff
@@ -450,6 +450,15 @@ static int run_connect(struct ov_private_data *data,
 	const char *option;
 	int err = 0;
 
+	/*
+	 * Unless set explicitly by the user default to 10 auth errors after a
+	 * succesful connection attempt was made before clearing them via VPN
+	 * agent.
+	 */
+	option = vpn_provider_get_string(provider, "AuthErrorLimit");
+	if (!option)
+		vpn_provider_set_string(provider, "AuthErrorLimit", 10);
```

Comment:
Should probably be "10".

Reply:
Oops, I guess I missed also the gcc warning as well. Fixed.